schema/elements.go

```diff
@@ -180,17 +184,45 @@ type Map struct {
 type Untyped struct {
 	// ElementRelationship states the relationship between the items, if
 	// container-typed data happens to be present here.
+	// * `deduced` implies that the behavior is based on the type of data.
```

Comment:
Can we get a slightly better explanation of what that means? Sounds like "This does the right thing in the right scenario" :-))

Reply:
I put it below https://github.com/kubernetes-sigs/structured-merge-diff/pull/71/files/914b72e3c5ebdf4f5a63c4fef4235a87f1b74fa6#diff-60731b77d91302b54bc9fb3b89f6521aR197

Should I just move that here?